tensorflow/compiler/tf2tensorrt/kernels/trt_engine_op.cc

```diff
@@ -721,7 +730,11 @@ void TRTEngineOp::ExecuteCalibration(OpKernelContext* ctx,
       VLOG(2) << "Passed calibration data";
     }
   }
-  ExecuteNativeSegment(ctx, async_helper);
+  if(!native_segment_absent_) {
+    ExecuteNativeSegment(ctx, async_helper);
+  } else{
+    LOG(ERROR)<< "Re-run calibration with original saved model graph, native segment not found";
```

Comment:
I would change the wording. We use the converter object to run calibration, and that always has the original graph. Calibration is not something that we can re-run with a different graph: we change the saved model only when we save the converted graph, but we cannot save the model without calibration.

```suggestion
    LOG(ERROR)<< "Calibration requires native segment, but it is not available in in the graph";
```